bin/collator/src/command.rs

```diff
@@ -819,16 +819,23 @@ pub fn run() -> Result<()> {
             let runner = cli.create_runner(&cli.run.normalize())?;
             let collator_options = cli.run.collator_options();
 
+            #[cfg(not(feature = "evm-tracing"))]
+            let additional_config = AdditionalConfig {
+                enable_evm_rpc: cli.enable_evm_rpc,
+                proposer_block_size_limit: cli.proposer_block_size_limit,
+                proposer_soft_deadline_percent: cli.proposer_soft_deadline_percent,
+            };
```

Comment:
Do you need to declare this _twice_?

What about this:
```
let additional_config = AdditionalConfig {
    #[cfg(feature = "evm-tracing")]
    evm_tracing_config: evm_tracing_config,
    enable_evm_rpc: cli.enable_evm_rpc,
    proposer_block_size_limit: cli.proposer_block_size_limit,
    proposer_soft_deadline_percent: cli.proposer_soft_deadline_percent,
};
```

Reply:
Good suggestion.
will do.